WcaOnRails/Gemfile

```diff
@@ -105,7 +105,7 @@ end
 
 group :development do
   gem 'overcommit', require: false
-  # RuboCop only supports Ruby 2.5 until 0.81
+  # RuboCop only supports our infrastructure until 0.81
```

Comment:
What is the constraint preventing us from using a more recent version of rubocop?

Reply:
Stupid runtime errors that I don't have the time to debug right now.